README.md

```diff
@@ -23,7 +23,15 @@ To setup the PatternFly 4 development environment:
 
 - clone the project
 - run `npm install` from the project root
-- run `npm run dev:lite`
+- run `npm run build-patternfly`
+- run `npm run dev`
+- open your browser to `http://localhost:8000`
+
+As an alternative development environment:
```

Comment:
Instead of listing this- we should be clear about why people would use this instead of dev above - seems odd to recommend two alternate ways of running the project.

Reply:
decided to remove the second dev configuration from the readme in favor of communicating that to users one on one.